src/main/resources/test_files/runtime/FunctionUserDefinedDynamic/NamedFunctionRef-Error10.jq

```diff
@@ -0,0 +1,9 @@
+(:JIQS: ShouldCrash; ErrorCode="XPST0017"; ErrorMetadata="LINE:5:COLUMN:10:":)
+declare function f($x) {
+$x.guess eq "English"
+};
+for $o in json-lines("... language test ...")
+where f($o)
+return $o
+
+(: referencing non-existing function in a FLWOR clause :)
```

Comment:
This is actually supposed to work -- any function declared in the prolog can be used everywhere in the query.